src/scripts/index.js

```diff
@@ -1,10 +1,502 @@
 import '../styles/main.css';
-import Button from './Button';
-import CallToAction from './CallToAction';
+// import Button from './Button';
+// import CallToAction from './CallToAction';
 
-const button = new Button(document.querySelector('.button'));
-const cta = new CallToAction(document.querySelector('.button-cta'));
+// const button = new Button(document.querySelector('.button'));
+// const cta = new CallToAction(document.querySelector('.button-cta'));
+
+// button.registerEvents();
+// cta.registerEvents();
+// cta.changeBgColor();
+
+
+// const toDoColumn = document.querySelector('.toDo');
+
+// if (toDoColumn !== null){
+//   toDoColumn.addEventListener('dblclick', function() {
+//     console.log(toDoColumn);
+//     console.log('apelez fct openmodal');
+//     openModal();
+//   });
+
+// }
+
+const modal = document.getElementById('modal');
+
+/**
+  * Open Modal
+  *
+*/
+function openModal() {
+  modal.style.display = 'block';
+}
+
+const closeIcon = document.querySelector('.close');
+closeIcon.addEventListener('click', function() {
+  modal.style.display = 'none';
+});
+
+
+const ticketNameInp = document.getElementById('ticketName');
+const descriptionInp = document.getElementById('description');
+
+let ticketID;
+let ticketName;
+let description;
+
+/**
+  * Save Tickets in localStorage
+  *
+*/
+function saveTicket() {
+  let tickets = [];
+  ticketName = ticketNameInp.value;
+  description = descriptionInp.value;
+
+  let id = 1;
+  const idFromLocalStorage = JSON.parse(localStorage.getItem('JiraID'));
+  if (idFromLocalStorage !== null) {
+    id = idFromLocalStorage + 1;
+  }
+
+  localStorage.setItem('JiraID', JSON.stringify(id));
+
+  ticketID = `JIRA-${id}`;
+  const obj =
+    {name: ticketName, description: description, id: ticketID, status: 'To Do'};
+  const objInLocalStorage = JSON.parse(localStorage.getItem('JiraTickets'));
+
+
+  if (objInLocalStorage !== null) {
+    tickets = objInLocalStorage;
+  }
+
+
+  tickets.push(obj);
+  localStorage.setItem('JiraTickets', JSON.stringify(tickets));
+  // displayTicket(ticketID, ticketName, description)
+}
+
+const saveBtn = document.getElementById('saveTicket');
+
+saveBtn.onclick = function() {
+  saveTicket();
+};
+
+/**
+  * Display Tickets from localstorage
+  *
+*/
+function displayTickets() {
+  const objInLocalStorage = JSON.parse(localStorage.getItem('JiraTickets'));
+
+
+  if (objInLocalStorage !== null) {
```

Comment:
Please break down the code in the for loop. Use spacing properly, respect the style guide.